definitions/divorce/json/CaseEvent/CaseEvent-pet-amend-nonprod.json

```diff
@@ -6,7 +6,7 @@
       "Name": "Allow Petitioner to amend",
       "Description": "Allow Petitioner to amend",
       "DisplayOrder": 23,
-      "PreConditionState(s)": "AwaitingHWFDecision;Submitted;Issued;Rejected;PendingRejection;solicitorAwaitingPaymentConfirmation;AosAwaiting;AosStarted;AosSubmittedAwaitingAnswer;AwaitingDecreeNisi;AosOverdue;AwaitingReissue;AosCompleted;AosAwaitingSol;AosPreSubmit;AosDrafted;AwaitingService",
+      "PreConditionState(s)": "Issued;Rejected;PendingRejection;AosAwaiting;AosStarted;AosSubmittedAwaitingAnswer;AwaitingDecreeNisi;AosOverdue;AwaitingReissue;AosCompleted;AosAwaitingSol;AosPreSubmit;AosDrafted;AwaitingService",
       "PostConditionState": "AwaitingAmendCase",
```

Comment:
Removing the pre-states that where causing some issues with the implementation of DIV-6193 to error